TreeDB/nativewire/dense_vector_search_test.go

```
@@ -0,0 +1,458 @@
+	}
+	mgr := collections.NewCollectionManager(db)
+	svc := documentservice.New(mgr)
+	routeProvider := &routingClusterSubmitter{fakeClusterSubmitter: &fakeClusterSubmitter{}}
+	server := NewServer(ServerOptions{Collections: mgr, Backend: db, ClusterSubmitter: routeProvider, DocumentService: svc})
+	t.Cleanup(func() { _ = server.Close(); _ = svc.Close(); _ = db.Close() })
+	ctx := context.Background()
+	client, cleanup, err := NewInProcessClient(ctx, server)
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer cleanup()
```

golangci-lint (2.13.2)

[error] Error return value is not checked

(errcheck)